omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/QuickVisitFragmentControllerTest.java

```diff
@@ -98,13 +103,43 @@ public void shouldFailIfActiveVisitAlreadyExists() throws Exception {
         HttpServletRequest request = mock(HttpServletRequest.class);
         when(request.getSession()).thenReturn(session);
 
-        FragmentActionResult result = controller.create(adtService, patient, visitLocation,
-                uiUtils, emrContext, request);
+        FragmentActionResult result = controller.create(adtService, patient, visitLocation, uiUtils, null, emrContext, request);
 
         assertThat(result, is(instanceOf(FailureResult.class)));
         verify(adtService, never()).ensureVisit(eq(patient), any(Date.class), eq(visitLocation));
     }
 
+    @Test
+    public void shouldCreateNewVisitWithVisitType() throws Exception {
+	    Patient patient = new Patient();
+	    Location visitLocation = new Location();
+
+	    String successMessage = "Success message";
+	    String formattedPatient = "Patient name";
+	    when(uiUtils.format(patient)).thenReturn(formattedPatient);
+	    when(uiUtils.message("coreapps.visit.createQuickVisit.successMessage", formattedPatient)).thenReturn(successMessage);
+
```

Comment:
I see lots of changes in this pull request and hence would expect a bit more than just one or two tests. :)